src/styles/main.scss

```diff
@@ -1,3 +1,259 @@
+$color-main: #060B35;
+$color-second: #616070;
+$color-blue: #00ACDC;
+$star: 4;
+
+.stars {
+  display: flex;
+  gap: 4px;
+
+  &__star {
+    background: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    width: 16px;
+    height: 16px;
+  }
+}
+
+@for $star from 1 through 4 {
+  .stars--#{$star} {
+    :nth-child(-n + #{$star}) {
+      background-image: url('../images/star-active.svg');
+    }
+  }
+}
+
+
+
 body {
+  font-family: Roboto, "Times New Roman", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
   margin: 0;
 }
+
+
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  position: relative;
+  padding: 0 50px;
+
+  box-shadow: 0 2px 4px 0 #0000000d;
+  &__logo {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+}
+
+.nav {
+
+  &__list {
+    display: flex;
+
+    margin: 0;
+    list-style-type: none;
+    padding-left: 0;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+
+    font-size: 12px;
+    font-weight: 500;
+    text-transform: uppercase;
+    color: #060B35;
+    text-decoration: none;
+    height: 60px;
+
+    transition: 0.3s;
+  }
+
+  &__link:hover {
+    color: $color-blue;
+  }
+
+}
+
+.is-active {
+  color: $color-blue;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+
+  position: absolute;
+  bottom: 0;
+
+  border-radius: 8px;
+  background-color: $color-blue;
+  width: 100%;
+  height: 4px;
+}
+
+.catalog {
+  --columns: 1;
```

Comment:
Why need to use  CSS variable in this place, not SCSS?

Reply:
Misha explained in class that it's better that way